docs/dev/table/functions/index.md

```diff
@@ -84,11 +84,13 @@ The resolution order is:
 1. Temporary catalog function
 2. Catalog function
 
-## Ambiguous Function Reference
+### Ambiguous Function Reference
 
 The resolution order is:
 
 1. Temporary system function
-2. System function
-3. Temporary catalog function, in the current catalog and current database of the session
-4. Catalog function, in the current catalog and current database of the session
+2. Temporary catalog function, in the current catalog and current database of the session
```

Comment:
I think the solution should still maintain the function resolution order. Changing it would fundamentally change the behaviors, and would plant trouble for future releases.

Reply:
I'm also wary about this change. IMHO function resolution order change can break user programs, so we probably need to have a discussion on the mailing list. Besides, I don't see how Module plays its part in the resolution. Is it treated as the `System function`?

Reply:
I agree we should have the resolution order as defined in FLIP-57. Unfortunately as I understand the state of the implementation we have an order as follows:
1. Temporary system functions
2. Modules system functions
3. Temporary catalog functions
4. Catalog functions
5. Built-in functions

Reply:
@dawidwys is right, actually we have three kinds of system functions:
- temporary system functions
- Module system functions: (CoreModule) built-in system functions, now it's actually planner related.
- Module system functions: user define module functions